### DISCUSSION TITLE
Room insert 함수 따로 구현에 대해서

### DISCUSSION THREAD

**Discussion by eunzinn** [euphony-io/check-in-listener#17] [Q&A] [answered]
https://gus0000123.medium.com/mvvm-aac-room-%EC%82%AC%EC%9A%A9%EB%B2%95-3-%EC%8B%A4%EC%A0%84%ED%8E%B8-f42598eaebb7

Insert나 Delete 할 때 함수를 따로 만드는 것 같은데 뭔지 궁금합니다!

**Comment by Jsueeee** [comment, marked as answer]
#15 여기서 MainActivity 쪽 확인해보시면 될 것 같습니다 :)
